.travis.yml

```diff
@@ -1,7 +1,14 @@
 language: node_js
 node_js:
-  - node
-  - 8
+    # Fix Node 16 version to 16.2.0
+    # 16.3.0 is causing an issue with mock-fs library
+    # causing some unit tests in cauldron API to fail
+    # This is a temporary solution while waiting for
+    # this issue to be addressed by Node team in a new
+    # release and/or mock-fs library
+    # Reference : https://github.com/tschaub/mock-fs/issues/332
+  - 16.2.0
```

Comment:
Would like to get your thoughts on this, but `mock-fs` is the only reason this needs to be pinned. Alternatives make testing more complicated.